addons/mail/static/src/chatter/web/form_renderer.js

```diff
@@ -43,4 +44,22 @@ patch(FormRenderer.prototype, {
         });
         return this.messagingState.thread.attachmentsInWebClientView.length > 0;
     },
+    mailLayout(hasAttachmentContainer) {
+        const xxl = this.uiService.size >= SIZES.XXL;
```

Comment:
`XXL` is the max, so `=== SIZES.XXL` is enough

Reply:
But what if we add `SIZES.XXXL`?

Reply:
> But what if we add `SIZES.XXXL`?

Probably we could add `ui.isHuge`?
Not a huge deal regardless (pun intended)